src/checkbox-group.ts

```diff
@@ -595,6 +594,10 @@ export default class GlideCoreCheckboxGroup
   }
 
   #onDefaultSlotChange() {
+    for (const checkbox of this.#checkboxElements) {
+      checkbox.privateVariant = 'minimal';
+    }
```

Comment:
```js
for (const checkbox of this.#checkboxElements) {
  if (checkbox.privateVariant !== 'minimal') {
    checkbox.privateVariant = 'minimal';
  }
}
```

I considered above at first to cut down on setting the property if it's already set.  But probably a premature optimization. What do you think? Do you like what is already in this PR or the change in the comment?

Reply:
> What do you think? 

Agreed. It's also not obvious what we'd be prematurely optimizing for.